capa/ida/helpers.py

```diff
@@ -191,3 +198,33 @@ def read(self, size):
 
     def close(self):
         return
+
+
+def save_results(resdoc, ruleset):
+    n = netnode.Netnode(CAPA_NETNODE)
+    n[NETNODE_RESULTS] = resdoc.json()
+    n[NETNODE_RULESET] = base64.b64encode(pickle.dumps(ruleset)).decode("ascii")
```

Comment:
we may want to introduce a version here, in the payload blobs, or even better, in the netnode path, so that we if we introduce a new format for the cached data, we don't run into trouble.